steps/versionbox-deploy/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: versionbox-deploy
+summary: |
+  Send your generated applications to VersionBox.
+description: "Send your generated ipa/apk files to VersionBox, automatically upload
+  a new version to your VersionBox application.\nTo use this step, you must have the
+  absolute path of the IPA/APK file generated on current build. \n"
+website: https://github.com/loopdigital/bitrise-step-versionbox-deploy
+source_code_url: https://github.com/loopdigital/bitrise-step-versionbox-deploy
+support_url: https://github.com/loopdigital/bitrise-step-versionbox-deploy/issues
+published_at: 2018-08-06T11:22:35.637122012+03:00
+source:
+  git: https://github.com/loopdigital/bitrise-step-versionbox-deploy.git
+  commit: 4868c5013e41eea5d1c4a0a7623b4b786a1d40c8
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
does the step use `git` and `wget` ?

Reply:
No, sorry for that.
But it uses ruby, does ruby exists on VM by default, or should I have to add it under deps?

Thanks.

Reply:
Hi, we have ruby installed on our vms, the current versions / stacks can be found here: https://github.com/bitrise-io/bitrise.io/tree/master/system_reports